src/lldpd-structs.h

```diff
@@ -275,6 +275,7 @@ MARSHAL_END(lldpd_port);
 /* Used to modify some port related settings */
 struct lldpd_port_set {
 	char *ifname;
+	char *p_id;
```

Comment:
I would call this new variable `local_id` as we expect a NULL-terminated string and we assume this is a local ID. Otherwise, it may be confusing in the future.